File: java/program/Petshop.java
// Kelas Petshop yang merepresentasikan entitas barang di petshop
class Petshop {
    private String id;        // ID unik dari barang
    private String nama;      // Nama barang
    private String kategori;  // Kategori barang (misal: Makanan, Alat Pembersih, dll.)
    private String harga;     // Harga barang dalam bentuk string

    // Konstruktor default tanpa parameter
    Petshop() {
    }

    // Konstruktor dengan parameter untuk inisialisasi objek langsung
    public Petshop(String id, String nama, String kategori, String harga) {
        this.id = id;
        this.nama = nama;
        this.kategori = kategori;
        this.harga = harga;
    }

    // Getter untuk mengambil nilai ID
    public String getId() {
        return id;
    }

    // Setter untuk mengatur nilai ID
    public void setId(String id) {
        this.id = id;
    }

    // Getter untuk mengambil nama barang
    public String getNama() {
        return nama;
    }

    // Setter untuk mengatur nama barang
    public void setNama(String nama) {
        this.nama = nama;
    }

    // Getter untuk mengambil kategori barang
    public String getKategori() {
        return kategori;
    }

    // Setter untuk mengatur kategori barang
    public void setKategori(String kategori) {
        this.kategori = kategori;
    }

    // Getter untuk mengambil harga barang
    public String getHarga() {
        return harga;
    }

    // Setter untuk mengatur harga barang
    public void setHarga(String harga) {
        this.harga = harga;
    }
}

File: python/program/Petshop.py
class Petshop:
    def __init__(self, id="", nama="", kategori="", harga=""):
        self.__id = id
        self.__nama = nama
        self.__kategori = kategori
        self.__harga = harga

    # Getter dan Setter untuk id
    def get_id(self):
        return self.__id

    def set_id(self, id):
        self.__id = id

    # Getter dan Setter untuk nama
    def get_nama(self):
        return self.__nama

    def set_nama(self, nama):
        self.__nama = nama

    # Getter dan Setter untuk kategori
    def get_kategori(self):
        return self.__kategori

    def set_kategori(self, kategori):
        self.__kategori = kategori

    # Getter dan Setter untuk harga
    def get_harga(self):
        return self.__harga

    def set_harga(self, harga):
        self.__harga = harga
